app.py

```
@@ -375,8 +375,9 @@ def chat_interface():
 
     # Get messages for the current chat
     messages = conversation_manager.get_context(chat_id)
+    context = Chat.get_context(chat_id)
```

Ruff (0.8.2)

Undefined name `Chat`

(F821)